_posts/2021-10-26-bwc-testing-for-opensearch.markdown

```diff
@@ -0,0 +1,94 @@
+---
+layout: post
+title:  "Backwards Compatibility Testing for OpenSearch"
+authors: 
+  - vachshah
+  - vemsarat
+date: 2021-10-26 01:01:01 -0700
+categories: 
+  - technical-post
+twittercard:
+  description: "This post provides details on the framework in OpenSearch for backwards compatibility testing"
```

Comment:
Missing period, it's a full sentence.

Reply:
@VachaShah Can you address @dblock's points above before lunch today (2021-11-05)? I'd love to get this out the door today.

Reply:
@stockholmux , @VachaShah is out today. 
I don't know if I have enough permissions to make changes on her fork. 

May be can publish this and then tweak it next week?

Reply:
@saratvemulapalli OK. Let's do that!

Reply:
Thank you @stockholmux @saratvemulapalli @dblock ! I will put up a PR for the tweak this week.

Reply:
@dblock I have created a PR #478 with the updates. Looks like opensearch-build#90 covers most of the bwc automation part, I added a little more along with it. LMK if I missed something.